src/pokemon-service/presenter/pokemon_test.go

```diff
@@ -0,0 +1,41 @@
+package presenter
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/GerardoHP/ondemand-go-bootcamp/entity"
+)
+
+func TestResponsePresenter(t *testing.T) {
+	pokemonName := "Pikachu"
+	pokemons := []*entity.Pokemon{
+		{ID: 1, Name: pokemonName, Image: "image", Url: "url"},
```

Comment:
Declare a pokemon struct to avoid repeating lines, you can include the Pikachu inside this struct and delete lines 11 and 27

Reply:
Added the pokemon name as a const, and since the structs can't be declared as const I created a generic method to retrieve the same struct as a comment before